backend/src/Controllers/getElectionResultsController.ts

```diff
@@ -22,31 +22,32 @@ const getElectionResults = async (req: any, res: any, next: any) => {
     }
 
     const election = req.election
-    const candidateNames = election.races[0].candidates.map((Candidate: any) => (Candidate.candidate_name))
-    const cvr = ballots.map((ballot: Ballot) => (
-        ballot.votes[0].scores.map((score: Score) => (
-            score.score
+    let results = []
+    for (let race_index = 0; race_index < election.races.length; race_index++) {
+        const candidateNames = election.races[race_index].candidates.map((Candidate: any) => (Candidate.candidate_name))
+        const cvr = ballots.map((ballot: Ballot) => (
```

Comment:
what does cvr stand for?

Reply:
Tests and validation added. 
CVR stands for cast vote record, was inherited from Jay's old code.